From ac4d123a1f92b9a2752ac9cd0c2be51d0c897a98 Mon Sep 17 00:00:00 2001
From: Joe Zhang <joe.zhang@technexion.com>
Date: Tue, 14 Jan 2025 14:32:54 +0800
Subject: [PATCH 1/3] ARM64: configs: tn_imx8_defconfig: add DRM_ITE_IT6263 for
 imx8mpevk

If not, the drm pipeline will fail to connect.
---
 arch/arm64/configs/tn_imx8_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/tn_imx8_defconfig b/arch/arm64/configs/tn_imx8_defconfig
index 9703ddefc0ec..7532e5fc8396 100644
--- a/arch/arm64/configs/tn_imx8_defconfig
+++ b/arch/arm64/configs/tn_imx8_defconfig
@@ -787,6 +787,7 @@ CONFIG_DRM_DW_HDMI_AHB_AUDIO=m
 CONFIG_DRM_DW_HDMI_I2S_AUDIO=m
 CONFIG_DRM_DW_HDMI_GP_AUDIO=y
 CONFIG_DRM_DW_HDMI_CEC=m
+CONFIG_DRM_ITE_IT6263=y
 CONFIG_DRM_ITE_IT6161=y
 CONFIG_DRM_TI_SN65DSI84=y
 CONFIG_DRM_IMX=y
-- 
2.34.1

